db/migrate/20170828034650_change_column_name.rb

```diff
@@ -0,0 +1,5 @@
+	class ChangeColumnName < ActiveRecord::Migration[5.1]
```

Comment:
You could have done a `rails db:rollback` and fixed the typo in the previous migration. Its all possible before you made a commit.